src/plugins/irc-events/link.js

```diff
@@ -159,7 +159,14 @@ function fetch(uri, cb) {
 	var limit = Helper.config.prefetchMaxImageSize * 1024;
 	req
 		.on("response", function(res) {
-			if (!(/^image\/.+/.test(res.headers["content-type"]))) {
+			if ((/^image\/.+/.test(res.headers["content-type"]))) {
+				// response is an image
+				// if Content-Length header reports a size exceeding the prefetch limit, abort fetch
+				const contentLength = parseInt(res.headers["content-length"], 10) || NaN;
```

Comment:
Do the 0 as I suggested.

Reply:
This and the other comments in the batch are done.